### ISSUE TITLE
New Script ABAWD Screening tool (input?)

### ISSUE THREAD

**Issue by CDPotter** [MN-Script-Team/DHS-MAXIS-Scripts#1208]
Sometime last year we built a screening tool to count a client's ABAWD months. @IlseFerris had used it before she left. 

It worked by selecting a member number/case number on a dialog, checking that member's counted months, then give a series of 2 dialogs with checkboxes for WREG exemptions and then ABAWD exemptions. It will case note any exemptions checked and follows a flowchart approach (So if someone is WREG exempt then it lists them as ABAWD exempt). It will then case note the results for that member number.

It is very antiquated as @RobertFewins-Kalb and I built it when we first started with scripts last summer. So it need some beefing up with functions and issue prevention. Since it's not just a straight upload I wanted to get people's thoughts on the script and if they think it would be a useful tool.

**Comment by IlseFerris** [comment]
I found it to be very helpful, especially as at the time I was very new to dealing with ABAWD exclusions and counted months.

**Comment by theVKC** [comment]
It has my support. @ruthmaloney, any thoughts from the ME/ABAWD side?

**Comment by CDPotter** [comment]
Digging through this script that we built over a year ago really help drive the point home that commenting is a MUST!

**Comment by IlseFerris** [comment]
@theVKC  = vindicated

**Comment by ruthmaloney** [comment]
I like it! And we need it due to the “clarifications” we’ve received since January. Also, it would be nice to have something for counties using “banked” ABAWD months for tracking. I’ve reviewed two counties these past couple of months that were using banked months and it was very confusing, hard to track, and caused issues for the counties surrounding these because of the lack of documentation.

Is there a way someone can send me some info on the flowchart used?

Ruth Maloney

**Comment by theVKC** [comment]
Per @CDPotter, this is done. Just waiting on script freeze to end. Also, please add a comment with the manual time to process for agencies estimating time savings. Thank you! :)
